src/metabase/models.clj

```diff
@@ -138,3 +143,33 @@
  [timeline-event TimelineEvent]
  [user User]
  [view-log ViewLog])
+
+(defenterprise resolve-enterprise-model
+  "OSS version; no-op."
+  metabase-enterprise.models
+  [x]
+  x)
+
+(methodical/defmethod t2.model/resolve-model :before :default
+  "Ensure the namespace for given model is loaded.
+  This is a safety mechanism as we moving to toucan2 and we don't need to require the model namespaces in order to use it."
+  [x]
+  (when (and (keyword? x)
+             (= (namespace x) "model"))
```

Comment:
why do we remove `(not (isa? x :metabase/model))` ? it was there to make sure we don't load the ns every time we call `resolve-model` if it's already loaded.

Reply:
You're right, for some reason I thought that wasn't necessary but I misunderstood its purpose. I'll add it back.